model/src/main/kotlin/licenses/LicenseConfiguration.kt

```diff
@@ -65,6 +70,11 @@ data class LicenseConfiguration(
         }
     }
 
+    /** A property allowing convenient access to the names of all categories defined. */
+    val categoryNames: SortedSet<String> by lazy {
+        categories.map { it.name }.toSortedSet()
```

Comment:
We could avoid one collection conversion here by using `categories.mapTo(sortedSetOf()) { it.name }`.